<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LTIN394677" failures="1" tests="1" name="EdgeTests" time="29.049" errors="0" timestamp="2023-02-23T12:14:07 IST">
  <testcase classname="testCases.TC_01_AccountRegistrationTest" name="test_account_registration" time="7.159">
    <failure type="java.lang.AssertionError" message="Test failed expected [Thank you for registering with Fake Online Clothing Store.] but found []">
      <![CDATA[java.lang.AssertionError: Test failed expected [Thank you for registering with Fake Online Clothing Store.] but found []
at testCases.TC_01_AccountRegistrationTest.test_account_registration(TC_01_AccountRegistrationTest.java:41)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1589)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- test_account_registration -->
</testsuite> <!-- EdgeTests -->
